model/Concept.php

```diff
@@ -662,9 +662,27 @@ public function removeDuplicatePropertyValues($ret, $duplicates)
 
             }
         }
+        // the logic is inverted for skos:prefLabel; remove the more specific property (see #854)
```

Comment:
The comment seems misleading; the logic is the same for skos:prefLabel as for other properties (remove the generic property, keep the specific one), but this special case code for skos:prefLabel is required because it's listed in DELETED_PROPERTIES and thus not handled the same way as other properties.